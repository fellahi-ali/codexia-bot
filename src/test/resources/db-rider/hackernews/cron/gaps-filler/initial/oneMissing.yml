hackernews_item:
    -
        external_id: 1
        type: story
        by: test1author
        title: Title for test1
        url: test 1 url
        time: 2009-01-15 05:56:17
        deleted: false
    -
        external_id: 3
        type: story
        by: test3author
        title: Title for test3
        url: test 3 url
        time: 2009-01-15 05:58:12
        deleted: false
