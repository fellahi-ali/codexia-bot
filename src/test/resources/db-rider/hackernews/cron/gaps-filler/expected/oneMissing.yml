hackernews_item:
    -
        external_id: 1
        type: story
        by: test1author
        title: Title for test1
        url: test 1 url
        time: 2009-01-15 05:56:17
        deleted: false
    -
        external_id: 3
        type: story
        by: test3author
        title: Title for test3
        url: test 3 url
        time: 2009-01-15 05:58:12
        deleted: false
    -
        external_id: 2
        type: story
        by: test2author
        title: Title for test2
        url: test 2 url
        time: 2009-01-15 06:12:49
        deleted: false
